src/app/products/manage-tax-components/view-tax-component/view-tax-component.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="container m-b-20" fxLayout="row" fxLayout.lt-md="column" fxLayoutAlign="end" fxLayoutGap="2%">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+</div>
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" class="content">
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Name
+        </div>
+
+        <div fxFlex="50%">
+          {{ taxComponentData.name }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Percentage
+        </div>
+
+        <div fxFlex="50%">
+          {{ taxComponentData.percentage }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
```

Comment:
Properties for debit account are missing, see the previous implementation here: https://github.com/openMF/community-app/blob/develop/app/views/products/tax/ViewTaxComponent.html
Update accordingly. Also, ensure you use ngIf (for optional properties) to only show those properties which have data.

Reply:
Credit account data is also optional. Please check again add the condition for that as well.